scones/app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,91 @@
+body {
```

Comment:
seems like you have really sconed your design skills based on how good this looks